pkg/frontend/operations_get.go

```diff
@@ -21,58 +21,72 @@ func (f *frontend) getOperations(w http.ResponseWriter, r *http.Request) {
 			{
 				Name: "Microsoft.RedHatOpenShift/locations/operationresults/read",
 				Display: api.Display{
-					Provider:  "Azure Red Hat OpenShift",
-					Resource:  "locations/operationresults",
-					Operation: "Read operation results",
+					Description: "Gets the status of an asynchronous operation result",
```

Comment:
I had left the descriptions blank because it's hard to write one that adds anything to what's already there.  If they *have* to be populated, can we just copy the Operation field verbatim for now?